### PR TITLE
(fix) fix NY slip op reporter info

### PR DESCRIPTION
**Pull request by ss108**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1,4 @@
-six
+attrs==21.4.0
+jsonschema==4.3.2
```
Comment (ss108) [inline requirements.txt:2]: I figured putting the test reqs in here made sense

**Comment by flooie** [comment]
Hey @ss108 What is your basis for changing the default citation here?

**Comment by flooie** [comment]
The slip search page, as well as the opinions that Ive come across never use the period.  I'm certainly no expert here.


Screen Shot 2022-06-29 at 11 42 17 AM
Screen Shot 2022-06-29 at 11 42 22 AM
Screen Shot 2022-06-29 at 11 41 53 AM
Screen Shot 2022-06-29 at 11 42 31 AM

File: reporters_db/data/reporters.json [hunk 2]
```diff
@@ -13627,6 +13627,25 @@
...
+        {
+            "cite_type": "state",
+            "editions": {
+                "N.Y. Slip Op.": {
+                    "end": null,
+                    "start": "1750-01-01T00:00:00"
+                }
+            },
+            "mlz_jurisdiction": [
+                "us:ny;court.appeals"
```
Comment (flooie) [inline reporters_db/data/reporters.json:13640]: This isn't your addition but, does the court of appeals actually issue slip opinions?

Comment (ss108) [reply reporters_db/data/reporters.json:13640]: Almost certainly not. I've never seen that. I haven't seen any intermediate appellate slip opinions either; I've only seen trial court opinions in NY as "slips".

**Comment by ss108** [comment]
@flooie 

See, e.g., https://casetext.com/case/wells-fargo-bank-469? (using "N.Y. Slip Op.") 

My belief that the version with periods is correct was founded on the normal way of abbreviating things in citations (for example, a Bluebook citation to a NY Court of Appeals opinion would not be correct if it ended with `(NY 1989)` as opposed to `(N.Y. 1989)`), as well as, to a lesser extent, what Casetext provides as the citation.

However, it appears you are correct: https://www.nycourts.gov/reporter/style-manual/2017/2017-SM.htm#2.2%20(a)%20(8)

(Note though that, unlike the case with California, where they take their custom style seriously, nobody in NY state courts seems to care what the official citation style is--not the lawyers, and not, from what I have seen, the judges)

I will change this to revert to the old default citation and add the version with periods as a variation

**Comment by flooie** [comment]
@ss108 

According to NY Courts website 

https://nycourts.gov/reporter/style-manual/2017/2017-SM.htm#2.2%20(a)%20(8)

```
(8) Citation to Slip Opinions
Opinions scheduled for publication in the Official Reports are cited as follows:
(People v Daly, — Misc 3d —, 2011 NY Slip Op 21371 [Crim Ct, NY County 2011])
(Franklin Corp. v Prahler, — AD3d —, 2011 NY Slip Op 07947 [4th Dept 2011])
(Tkeshelashvili v State of New York, — NY3d —, 2011 NY Slip Op 08451 [2011])
```

**Comment by flooie** [comment]
> @flooie
> 
> See, e.g., https://casetext.com/case/wells-fargo-bank-469? (using "N.Y. Slip Op.")
> 
> My belief that the version with periods is correct was founded on the normal way of abbreviating things in citations (for example, a Bluebook citation to a NY Court of Appeals opinion would not be correct if it ended with `(NY 1989)` as opposed to `(N.Y. 1989)`), as well as, to a lesser extent, what Casetext provides as the citation.
> 
> However, it appears you are correct: https://www.nycourts.gov/reporter/style-manual/2017/2017-SM.htm#2.2%20(a)%20(8)
> 
> (Note though that, unlike the case with California, where they take their custom style seriously, nobody in NY state courts seems to care what the official citation style is--not the lawyers, and not, from what I have seen, the judges)
> 
> I will change this to revert to the old default citation and add the version with periods as a variation

Thanks, sorry I responded before I saw your full comment.  Thanks for clarifying and yes, it also is very much an aberration in style.

**Comment by ss108** [comment]
@flooie 
NP, sorry I didn't check the official style guide first :sweat_smile:

File: reporters_db/data/reporters.json (hunk 2 above)
Comment (ss108) [reply reporters_db/data/reporters.json:13640]: I actually think NY CoA and AD opinions do initially come out as slip ops, but they are quickly "reported", and so realistically, one isn't going to encounter them cited as slip ops. https://casetext.com/case/arons-v-jutkowitz-3 (you can see it has a slip op number)

So ultimately, a citation to a Slip Op indicates the opinion is a trial court opinion with a very high degree of confidence.

**Comment by ss108** [comment]
@flooie updated

**Comment by flooie** [comment]
@ss108 I still think we should remove the non required python requirements from requirements.txt. I could be wrong but in theory that just makes the package larger for people who want to install the project.  Perhaps we could move them to a requirements-dev.txt file and note it in the readme.md

**Comment by flooie** [comment]
And do we really not need six anymore?

**Comment by ss108** [comment]
I manually created a `requirements.dev.txt` file and split the dependencies. I don't know enough about six to be able to say, sorry. 

I suppose it's being used for downstream users/consumers who are using Python 2?

**Comment by flooie** [comment]
I only asked about six - because I thought you removed it in the PR

**Comment by flooie** [comment]
@ss108 circling back to this today.  I think we can approve it once we set the default to NY Slip Op.  and point the variations towards it.  Thanks and sorry for letting this hang for so long.

**Comment by ss108** [comment]
@flooie Should be good now. `NY Slip Op` (no punctuation), as you guys originally had it, is correct, and this PR adds variations with punctuation.

**Comment by flooie** [comment]
Thanks @ss108

**Comment by mlissner** [comment]
Six will outlive us all.

**Pull request merged**